src/modules/calls/dto/call-query.dto.ts

```diff
@@ -1,33 +1,13 @@
 import { ApiProperty } from '@nestjs/swagger';
 
 export class CallQueryDto {
-  @ApiProperty({
```

Comment:
please leave this API property in, to define that the number is an integer for the app developers.